### PR TITLE
Terms delete

### PR DESCRIPTION
**Pull request by ColeLindner**
Finished creating the delete terms button.
Currently have it only test one Term, but should have ability to delete any term based on term name and having the button respond to the term it is by.

### CODE REVIEW

**Review by Pirundy — APPROVED** [review]
Looks good

File: views/layout/partials/scripts.ejs [hunk 1]
```diff
@@ -163,4 +163,22 @@
...
+<!-- Script with working on the terms page -->
+<script>
+  function showModal(termNumber){
+    const userName = document.getElementById(`userTerm${termNumber}`).innerText
+    const title = document.getElementById('modalTitle')
+    title.innerHTML = `Are you sure you want to remove Term '${userName}'`
+    $('#termsModal').modal('show')
+  }
+  function removeTerm(termNumber){
+    console.log(termNumber)
```
Comment (Jake-Derrick) [inline views/layout/partials/scripts.ejs]: Console.log should be removed

File: views/pages/terms/index.ejs [hunk 2]
```diff
@@ -2,9 +2,20 @@
     <div class="text-center bold">Terms</div>
 </div>
 
-<div class="card bg-primary" id="body">
-    <div class="text-center bold" contenteditable="true" id="Term1">Term 1</div>
+<div class="card bg-primary" id="Term1">
+    <div class="text-center bold" contenteditable="true" id="userTerm1">Term 1 </div> 
+    <div align="right">
```
Comment (Jake-Derrick) [inline views/pages/terms/index.ejs]: Align is deprecated, should use text-align

Inline styles should be style="text-align:right;"

**Review by Jake-Derrick — CHANGES_REQUESTED** [review]
Overall I think it looks a bit odd with the yes and no buttons spanning the entire page. I would recommend removing the modal and adding this: 
if (window.confirm('Are you sure you want to remove this term?') {
  // your termToBeRemoved code
}
And replacing the onClick with removeTerm(1)

This is a bit out of scope, but the page is set up weird. The structure should stay consistent with other pages. That would be
image

Additionally, there should be a unit test checking that the button rendered on the page

**Comment by presmiks18** [comment]
I think once the changes Jake suggested are implemented this looks good.

File: views/pages/terms/index.ejs (hunk 2 above)
Comment (ColeLindner) [reply views/pages/terms/index.ejs]: I have now made the changes that you have suggested.

**Review by Jake-Derrick — APPROVED** [review]
Looks good! Technically it should still have a unit test.

**Pull request merged**